content/matplotlib/concepts/pyplot/terms/bar/bar.md

```diff
@@ -0,0 +1,30 @@
+# .bar()
```

Comment:
This entry is missing the metadata section.  You can look at the [metadata section of the style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata) to see what is supposed to go into it

Also, remove this first header as the Title: in the metadata will provide it.